docs/userguide.md

```diff
@@ -31,7 +30,7 @@
   - [Cleanup](#cleanup)
     - [Deployment Cleanup](#deployment-cleanup)
```

Comment:
Is this section left as is because of the issue with [uninstalling the charts](https://github.com/kubernetes-sigs/federation-v2/blob/master/charts/federation-v2/README.md#uninstalling-the-chart)?

Reply:
I think make reference information for both install/uninstall is better. So, either of below is fine to me.
1.  make a clean up section to refer to unstallation the charts.
2. delete this section and make Helm Chart Deployment section as
```diff
- You can refer to [helm chart installation guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/charts/federation-v2/README.md)
- to install federation-v2.
+ You can refer to [helm chart guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/charts/federation-v2/README.md)
+ to install/uninstall federation-v2.
```

Reply:
@irfanurrehman Good catch, I just missed it.  I've removed the `Cleanup` section and added mention of uninstall to the helm section as per @xunpan's suggestion.